tools/cli/commands.go

```diff
@@ -559,7 +560,7 @@ func StartWorkflow(c *cli.Context) {
 	tcCtx, cancel := newContext()
 	defer cancel()
 
-	resp, err := serviceClient.StartWorkflowExecution(tcCtx, &s.StartWorkflowExecutionRequest{
+	startRequest := &s.StartWorkflowExecutionRequest{
```

Comment:
If you use the serverFrontendClient, then you don't have to update the client repo IDLs to have CLI. Refering this: https://github.com/uber/cadence/blob/master/tools/cli/commands.go#L1176